src/Executables/Examples/RandomAmr/RandomAmr.hpp

```diff
@@ -0,0 +1,93 @@
+// Distributed under the MIT License.
```

Comment:
Should this executable go in `tests/`?

Reply:
It could, but I thought it could be a good example of a parallel executable that can easily be run on multiple nodes and cores, and used in a tutorial.   This is not the final state of the executable, it will do full h-refinement.  I just added it now to test the reduction action.

Reply:
ok, I'm not sure if `Examples/` or `tests/` is the right place. Either seems fine I think. @wthrowe @nilsdeppe any opinions?